sdk/eventgrid/eventgrid/src/models.ts

```diff
@@ -153,6 +153,14 @@ export interface CloudEvent<T> {
    * Additional context attributes for the event. The Cloud Event specification refers to these as "extension attributes".
    */
   extensionAttributes?: Record<string, unknown>;
+  /**
+   * The version of the CloudEvents specification which the event uses.
+   */
+  specversion: string | "1.0";
```

Comment:
should this be optional as well?  Existing callers would not have this member present and so I think their call sites would no longer type check?  If it is not provided, we could assume 1.0?